README.md

```diff
@@ -46,25 +46,33 @@ Citus Cloud runs on top of AWS as a fully managed database as a service and has
 
 If you're looking to get started locally, you can follow the following steps to get up and running.
 
-* Install docker-compose: [Mac][mac_install] | [Linux][linux_install]
-* (Mac only) connect to Docker VM
-  ```bash
-  eval $(docker-machine env default)
-  ```
-
-* Pull and start the docker images
+1. Install Docker Community Edition and docker-compose
+  * Mac:
+    1. [Download](https://www.docker.com/community-edition#/download) and install Docker.
+    2. Start Docker by clicking on the application’s icon.
+  * Linux:
+    ```bash
+    curl -sSL https://get.docker.com/ | sudo bash
```

Comment:
[The script itself](https://get.docker.com) says to use `curl <url> | sh`. Can we do that instead of bash?